src/coreclr/jit/loopcloning.cpp

```diff
@@ -2622,6 +2621,7 @@ PhaseStatus Compiler::optCloneLoops()
     {
         JITDUMP("Recompute reachability and dominators after loop cloning\n");
         constexpr bool computePreds = false;
+        // TODO: recompute the loop table, to include the slow loop path in the table?
```

Comment:
I think you should seriously consider this -- eg the loop and its clone may have common hoistables.

Reply:
Also then the slow path would show up in the dot graph loop nesting hierarchy....

Reply:
I opened https://github.com/dotnet/runtime/issues/62615 to track this.